lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1009,6 +1015,11 @@ def _create_deployment_parameters(
         arm_parameters.storage_name = get_storage_account_name(
             self.subscription_id, arm_parameters.location
         )
+        if (
+            self._azure_runbook.availability_set_properties
```

Comment:
Because we hope to disable AS by default, remove the initialize of availability_set_properties. So it's empty by default.